mysql-test/suite/rocksdb.rpl/t/suite.opt

```diff
@@ -1 +1 @@
-$ROCKSDB_OPT $ROCKSDB_LOAD_ADD
+$ROCKSDB_OPT $ROCKSDB_LOAD_ADD --loose-rocksdb_unsafe_for_binlog=true
```

Comment:
Then perhaps this option should be added to individual .opt files for the unsafe tests? I am worried that mixed/stmt testing is going to be sub-standard here even for the subset of the testcases that don't have any inherent row format dependencies.

If you considered the above and still want a global option here, then it's OK